authors/apps/articles/views.py

```diff
@@ -244,3 +244,23 @@ def post(self, request, *args, **kwargs):
         article.save()
         serializer = self.serializer_class(article, context={'request':self.request})
         return Response(serializer.data, status=returned_status)
+
+class RatingsView(generics.ListCreateAPIView):
+    """
+    view for rating for articles
+    """
+    permission_classes = (IsAuthenticatedOrReadOnly,)
+    serializer_class = serializers.ArticleRatingSerializer
```

Comment:
@dannylwe Please add a queryset to this view inorder for the get method to work.